STYLE_GUIDE.md

```diff
@@ -6,10 +6,11 @@ number of errors, and promote consistency.
 
 ## TensorFlow Style
 
-Follow the [TensorFlow style
-guide](https://www.tensorflow.org/community/style_guide) and [documentation
-guide](https://www.tensorflow.org/community/documentation), and in particular
-the [TensorFlow code
+Follow the [TensorFlow style 
```

Comment:
Revert changes in this file?

Reply:
Are you sure? The old links were dead. These seem to be working, at least the last time I checked.

Reply:
Or would you want this in a separate PR? Can do that too.

Reply:
I think just neater if this is in a separate PR (thanks for catching this!)

Reply:
I dropped these changes from this PR. Will make a separate one for these.

Reply:
New PR: https://github.com/tensorflow/probability/pull/1332